lib/index.js

```diff
@@ -0,0 +1,8 @@
+
+const path = require('path');
```

Comment:
why is this located in lib directory?

Reply:
@derberg Sorry I used it for testing removed